scalafix/rules/src/main/scala/fix/FixBqSaveAsTable.scala

```diff
@@ -11,28 +11,43 @@ class FixBqSaveAsTable extends SemanticRule("FixBqSaveAsTable") {
 
   override def fix(implicit doc: SemanticDocument): Patch = {
     doc.tree.collect {
-      case a @ Term.Apply(fun, head :: tail) =>
+      case a @ Term.Apply(fun, params) =>
         if (fun.symbol.normalized.toString.contains(methodName)) {
```

Comment:
`methodName` and `scoll` should be defined as constants in companion object

Reply:
not sure how important it is in this case, these files are standalone scripts